broker/src/test/java/io/zeebe/broker/system/configuration/RocksdbCfgTest.java

```diff
@@ -25,7 +25,7 @@ public void shouldDisableStatisticsPerDefault() {
     final var rocksdb = cfg.getExperimental().getRocksdb();
 
     // then
-    assertThat(rocksdb.isStatisticsEnabled()).isFalse();
+    assertThat(rocksdb.isEnableStatistics()).isFalse();
```

Comment:
This looks a bit weird but yeah either that or the other right :D

Reply:
Yeah I thought so too - we could have the property be `enableStatistics` and the method still be `isStatisticsEnabled` (and same for WAL), but then IntelliJ will never generate it :sweat_smile: 

I don't have a strong opinion, I'm fine with either, so unless you insist I'll go with IntelliJ defaults (i.e. `isEnableStatistics`).

Reply:
yeah it is not beautiful but I can live with it :grin: